x/concentrated-liquidity/incentives_test.go

```diff
@@ -290,40 +290,42 @@ func (s *KeeperTestSuite) TestCreateAndGetUptimeAccumulators() {
 			expectedPass:        false,
 		},
 	}
+	s.runMultipleAuthorizedUptimes(func() {
```

Comment:
Note to reviewer: sorry about the horrible diff rendering here – it should be manageable with split diff review, but it seems the cases that were indented + had cleanup changes got jumbled a bit.